drivers/SmartThings/matter-rvc/src/init.lua

```diff
@@ -162,6 +162,8 @@ local function rvc_run_mode_supported_mode_attr_handler(driver, device, ib, resp
   )
 
   local component = device.profile.components["runMode"]
+  local labels = get_field_labels_of_supported_modes(device, RVC_RUN_MODE_SUPPORTED_MODES)
+  device:emit_component_event(component, capabilities.mode.supportedArguments(labels))
```

Comment:
Will `supportedArguments` be available for R2?

Reply:
It is set to device configuration to use supportedArguments in RVC.
https://github.ecodesamsung.com/iot/st-presentation/blob/master/device-configurations/EdgeGenericRVC.json